.tekton/pull-request.yaml

```diff
@@ -147,6 +147,31 @@ spec:
               image: registry.access.redhat.com/ubi9/python-39:latest
               workingDir: $(workspaces.workspace.path)
               script: make install_pipenv sync_pipenv lint_yamls
+      - name: run-grafana-dashboard-lint
+        workspaces:
+          - name: workspace
+            workspace: workspace
+        runAfter:
+          - clone-repository
+        taskSpec:
+          workspaces:
+            - name: workspace
+          steps:
+            - name: run-grafana-dashboard-lint
+              image: registry.access.redhat.com/ubi9/go-toolset:latest
+              workingDir: $(workspaces.workspace.path)
+              script: |
```

Comment:
Should this test run in local dev setups as well? If it is, it should probably be implemented inside the makefile rather than inside this file.

Reply:
Need to update the contributing guide if there are any changes to the requirements for running the tests locally.

Reply:
Done both stuff in the latest commit

Reply:
Please make it all on a single commit. See this section: https://github.com/redhat-appstudio/o11y/blob/main/CONTRIBUTING.md#pull-request-span

Reply:
Solved! But I think I'm triggering the dashboard-linter twice in the CI. I need to check on that